Microsoft.Toolkit.Uwp.UI.Controls.Core/DropShadowPanel/IAlphaMaskProvider.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Composition;
+
+/// <summary>
+/// Any user control can implement this interface to provide a custom alpha mask to it's parent <see cref="DropShadowPanel"/>
```

Comment:
Think there's no namespace the comment can't be resolved properly?

 D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Core\DropShadowPanel\IAlphaMaskProvider.cs(8,108): error CS1574: XML comment has cref attribute 'DropShadowPanel' that could not be resolved [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls.Core\Microsoft.Toolkit.Uwp.UI.Controls.Core.csproj]

Once moved though we won't be able to reference the control directly anyway, just in text.